backend/src/main/java/ch/puzzle/okr/controller/KeyResultController.java

```diff
@@ -33,7 +40,29 @@ public KeyResultController(KeyResultService keyResultService, KeyResultMapper ke
     })
     @PutMapping("/{id}")
     public ResponseEntity<KeyResult> updateKeyResult(@PathVariable long id, @RequestBody KeyResultDto keyResultDto) {
-            keyResultDto.setId(id);
-            return ResponseEntity.status(HttpStatus.OK).body(this.keyResultService.updateKeyResult(keyResultMapper.toKeyResult(keyResultDto)));
+        keyResultDto.setId(id);
+        return ResponseEntity.status(HttpStatus.OK).body(this.keyResultService.updateKeyResult(keyResultMapper.toKeyResult(keyResultDto)));
+    }
+
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Get all key results",
+                    content = {@Content(mediaType = "application/json",
+                            schema = @Schema(implementation = KeyResultDto.class))}),
+    })
+    @GetMapping
+    public List<KeyResult> getAllKeyResults() {
+        return (List<KeyResult>) keyResultRepository.findAll();
```

Comment:
Hier so wie in den anderen Controllern mit dem Service und Mapper. Damit wir eine Liste von Dto's bekommen